src/components/Revoke/RevokeForm.vue

```diff
@@ -9,18 +9,18 @@
       </div>
       <p class="text-black-400 text-lg" v-html="$t('revoke.message', { authority, username })" />
       <div
-        v-if="account && account.name && hasRequiredKey ===false"
+        v-if="account && account.name && !hasRequiredKey"
         class="alert alert-warning mt-4"
       >
-        {{ $t('authorize.requires_active_key') }}
+        {{ $t('authorize.requires_active_key', { authority }) }}
       </div>
     </div>
     <div class="mt-2">
       <router-link
-        v-if="!(account && account.name) || hasRequiredKey === false"
+        v-if="!(account && account.name) || !hasRequiredKey"
         :to="{
           name: 'login',
-          query: { redirect: $route.fullPath, authority: 'active' },
+          query: { redirect: $route.fullPath, authority },
```

Comment:
@dkildar Shouldn't here be `authority: 'active'` because revoke and authorize pages require active permission at minimum but it is not defined now with this change or is it?